Added spacing feature to internal blocks.

NBonaparte opened the pull request:
This is my effort at implementing the feature I requested (#46). It can be turned on or off by setting `spacing: true` or `spacing: false` in the (global) bar section of the config; it defaults to false if `spacing` is not set.

The commit also removed some trailing whitespace.

geommer commented:
Thank you @NBonaparte for the contribution, and I like the idea that spacing is boolean because I would have made it integer. But your solution is better. I just have one notice, wouldn't it be better if `spacing` was included in `struct blk_intern` instead of `ya_bar_t` to make it more configurable? Please also notice that this option affects only internal blocks, so a new user may think this option is not working if he adds it for an external block. Thus I think it is more relevant to move this variable (i.e. `spacing`) to the `struct blk_intern` structure. A rename to `internal-spacing` would be more descriptive and consistent with the other internal options' names in such case.

NBonaparte commented:
I have changed `spacing` to `internal-spacing` and moved it to `blk-intern` in 4ec639753efb58bc7de85079be8579c1737d5668. 
I would like to suggest that when this feature is documented in the README, it should be stated that it works only with monospace fonts (obvious, but important).

geommer commented:
That is perfect now, and I will mention about monospace in README.
